10_Download Time Calculator.py

```diff
@@ -1,32 +1,65 @@
 # Write a procedure download_time which takes as inputs a file size, the
 # units that file size is given in, bandwidth and the units for
-# bandwidth (excluding per second) and returns the time taken to download 
+# bandwidth (excluding per second) and returns the time taken to download
 # the file.
 # Your answer should be a string in the form
 # "<number> hours, <number> minutes, <number> seconds"
 
 # Some information you might find useful is the number of bits
 # in kilobits (kb), kilobytes (kB), megabits (Mb), megabytes (MB),
 # gigabits (Gb), gigabytes (GB) and terabits (Tb), terabytes (TB).
+#
+# print 2 ** 10      # one kilobit, kb
+# print 2 ** 10 * 8  # one kilobyte, kB
+#
+# print 2 ** 20      # one megabit, Mb
+# print 2 ** 20 * 8  # one megabyte, MB
+#
+# print 2 ** 30      # one gigabit, Gb
+# print 2 ** 30 * 8  # one gigabyte, GB
+#
+# print 2 ** 40      # one terabit, Tb
+# print 2 ** 40 * 8  # one terabyte, TB
 
-#print 2 ** 10      # one kilobit, kb
-#print 2 ** 10 * 8  # one kilobyte, kB
-
-#print 2 ** 20      # one megabit, Mb
-#print 2 ** 20 * 8  # one megabyte, MB
-
-#print 2 ** 30      # one gigabit, Gb
-#print 2 ** 30 * 8  # one gigabyte, GB
-
-#print 2 ** 40      # one terabit, Tb
-#print 2 ** 40 * 8  # one terabyte, TB
-
-# Often bandwidth is given in megabits (Mb) per second whereas file size 
+# Often bandwidth is given in megabits (Mb) per second whereas file size
 # is given in megabytes (MB).
 
-    
-def download_time(filesize, fileUnit, BandWidth, BandWiddthUnit):
-    # Your code
+
+def download_time(filesize, fileUnit, BandWidth, BandWiddthUnit):
+
+    bits={'kb':1024,'kB':8192,'Mb':1048576,'MB':8388608,'Gb':1073741824,
+            'GB':8589934592,'Tb':1099511627776,'TB':8796093022208}
+
+    filesize_bits=filesize*bits[fileUnit]
+    bandwidth_bits=BandWidth*bits[BandWiddthUnit]
+
+    # download time in seconds
+    download_time=filesize_bits/bandwidth_bits
+
+    #converting download_time into hour minute and seconds mark
+    hours=int(download_time/3600)
```

Comment:
Please check `d, m = divmod(x, y)`. You will not need redundant operation of `download_time % 3600`